site/docs/contribute/environment.md

```diff
@@ -42,5 +42,5 @@ a docker registry as a proxy for all public images. This registry runs in a dock
 
 ```shell
 export DOCKERHUB_USERNAME='your docker hub username'
-export DOCKERHUB_PASSWORD='your password'
+export DOCKERHUB_PASSWORD='your GIT user token'
```

Comment:
Please revert. These are variables for login to dockerhub. It has nothing to do with git.

Reply:
@roee88 I could not get anything to work when using my docker hub password, nor my docker hub user token.  Only when I linked the m4d organization from git with docker hub and used my git user token here did anything work.  Perhaps we need clearer instructions about docker hub and m4d organization in git?

Reply:
I'll let @froesef jump in because I don't understand what dockerhub has to so with github. Unless you misread DOCKERHUB_ as DOCKER?